src/components/Common/QRCodeDialog.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTranslation } from 'react-i18next'
+import React from 'react'
+import toast from 'react-hot-toast'
```

Comment:
Consider using [Snackbar & Toast](https://mui.com/material-ui/react-snackbar/) so `react-hot-toast` can be removed from dependencies